README.md

```diff
@@ -320,6 +320,7 @@ These selectors are applicable when "deviceType" is "netDevice" (note: this is d
 | "devices"      | N        | Target Devices' device Hex code as string                                | `string` list Default: `null`                       | "devices": ["154c", "1889", "1018"]                                                              |
 | "drivers"      | N        | Target device driver names as string                                     | `string` list Default: `null`                       | "drivers": ["vfio-pci"]                                                                          |
 | "pciAddresses" | N        | Target device's pci address as string                                    | `string` list Default: `null`                       | "pciAddresses": ["0000:03:02.0"]                                                                 |
+| "acpiIndexes"  | N        | Target device's acpi index as string                                     | `string` list Default: `null`                       | "acpiIndexes": ["101"]                                                                           |
```

Comment:
with how you implemented it today, this selector is available for auxiliay network devices  and accelerators as well.

acpiIndex selector should appear in all relevant sections.
(but see my previous comments first as i think we need to have this one for accelerator and network devices only for now)

Reply:
I think we should support this for network and accelerators I don't think there is a use-case for this on auxiliary devices

Reply:
Added to the accelerators section